substrate-node/pallets/pallet-tfgrid/src/farm.rs

```diff
@@ -60,11 +60,13 @@ impl<T: Config> Clone for FarmName<T> {
     }
 }
 
-pub fn replace_farm_name_spaces_with_underscores(input: &[u8]) -> Vec<u8> {
+pub fn replace_farm_name_invalid_characters(input: &[u8]) -> Vec<u8> {
     input
         .iter()
         .map(|c| match c {
             b' ' => b'_',
+            39 => b'-',
```

Comment:
change this to be `b'x'` where x is the character to replace, for readability purposes

Reply:
Escape the `'` you want to replace, i.e. `b'\''`